src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -869,21 +871,27 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             _renderEngine->SetRetroTerminalEffect(newAppearance->RetroTerminalEffect());
             _renderEngine->SetPixelShaderPath(newAppearance->PixelShaderPath());
 
+            // Skip Unfocused Opacity when EnableUnfocusedAcrylic is false
+            // and Acrylic is true as this results into seeing the opacity going
+            // from solid to unfocused to focused bug.
+            bool skipUnfocusedOpacity = !focused && !(!_settings->EnableUnfocusedAcrylic() && UseAcrylic());

```

Comment:
```suggestion
            bool skipUnfocusedOpacity = !focused && (_settings->EnableUnfocusedAcrylic() || !UseAcrylic());
```
Personally, I'd prefer that you [De Morgan's law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) it haha

Reply:
Should be good now